e2e-tests/README.md

```diff
@@ -0,0 +1,119 @@
+# End - End UI Tests
+### Overview
+The End - End UI framework is based on [Cucumber](https://github.com/cucumber-rs/cucumber "Cucumber") and [Thrityfour](https://github.com/stevepryde/thirtyfour/tree/main/thirtyfour "Thrityfour"). The tests are written in Gherkin format and they are executed with the aid of cucumber-rs. 
+
+For example, the tests under package e2e-tests can be run with the below command to test localhost. 
+(Make sure the localhost is up and running with the [podman](#Setting up Localhost))
+```shell
+git clone https://github.com/trustification/trustification
+cd e2e-tests 
```

Comment:
I'm still bothered by the implication that running the tests is easy as cloning the repo. Maybe just assume whoever is running them has already cloned the repo?
```suggestion
From the directory containing this README file, run the following:
```